patches/0006-Add-CNG-crypto-backend.patch

```diff
@@ -692,26 +692,26 @@ index 512e4a4d6616e5..284db01c221d93 100644
  		if err != nil {
  			return nil, err
 diff --git a/src/crypto/rsa/rsa_test.go b/src/crypto/rsa/rsa_test.go
-index 379fb06526ccc7..d844cc4b8d749e 100644
+index fb3f821e76a6d4..6972ccbf2fcbde 100644
 --- a/src/crypto/rsa/rsa_test.go
 +++ b/src/crypto/rsa/rsa_test.go
 @@ -8,6 +8,7 @@ import (
  	"bufio"
  	"bytes"
  	"crypto"
 +	boring "crypto/internal/backend"
+ 	"crypto/internal/boring"
```

Comment:
This line should be deleted and the previous line addition moved to Patch-003

Reply:
See what I did in the upstream sync PR: https://github.com/microsoft/go/commit/c244632c6df4af681bf5d0f2c14f5d3014a505d7